conformance/results/mypy/callables_kwargs.toml

```diff
@@ -1,4 +1,4 @@
-conformant = "Pass"
+conformant = "Partial"
```

Comment:
Please add a multiline "notes" entry that summarizes the test failures. See other test result files for examples. This should be done for the pyright results as well.